makefiles/amy-keyring.mk

```
@@ -6,22 +6,16 @@ SUBPROJECTS          += amy-keyring
 AMY_KEYRING_VERSION  := 2021.07.10
 DEB_AMY_KEYRING_V    ?= $(AMY_KEYRING_VERSION)
 
-ifneq ($(wildcard $(BUILD_STAGE)/amy-keyring/.build_complete),)
 amy-keyring:
-	@echo "Using previously built amy-keyring."
-else
-amy-keyring: setup
-	mkdir -p $(BUILD_STAGE)/amy-keyring/$(MEMO_PREFIX)/etc/apt/trusted.gpg.d
-	cp -a $(BUILD_MISC)/keyrings/anamy/anamy-repo.gpg $(BUILD_STAGE)/amy-keyring/$(MEMO_PREFIX)/etc/apt/trusted.gpg.d
-	$(call AFTER_BUILD)
-endif
+	@echo "amy-keyring do not need to be built."
```

I hate fixing typo..

```suggestion
	@echo "amy-keyring does not need to be built."
```